.github/workflows/sycl-precommit.yml

```diff
@@ -69,27 +65,38 @@ jobs:
           else
             echo 'arc_tests="Matrix/"' >> "$GITHUB_OUTPUT"
           fi
-  test:
-    needs: [build, detect_changes, determine_arc_tests]
-    if: ${{ always() && !cancelled() && needs.build.outputs.build_conclusion == 'success' }}
+
+  test_linux:
+    needs: [build_linux, detect_changes, determine_arc_tests]
+    if: ${{ always() && !cancelled() && needs.build_linux.outputs.build_conclusion == 'success' }}
     strategy:
       fail-fast: false
       matrix:
         include:
-          - name: AMD/HIP
+          # XXX CUDA testing is in a separate workflow because it runs on AWS
+          # which has different access restrictions/requirements. See
+          # sycl-linux-precommit.yml if you're looking for the CUDA precommit
+          # definition
+          - name: E2E on AMD/HIP
             runner: '["Linux", "amdgpu"]'
             image: ghcr.io/intel/llvm/ubuntu2204_build:latest-0300ac924620a51f76c4929794637b82790f12ab
             image_options: -u 1001 --device=/dev/dri --device=/dev/kfd
             target_devices: ext_oneapi_hip:gpu
-          - name: Intel
+
+          - name: E2E on Intel GEN12; level zero GPU, OpenCL CPU
             runner: '["Linux", "gen12"]'
             image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:latest
-            image_options: -u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN
             target_devices: ext_oneapi_level_zero:gpu;opencl:gpu;opencl:cpu
             reset_gpu: true
             install_drivers: ${{ contains(needs.detect_changes.outputs.filters, 'drivers') }}
             extra_lit_opts: --param gpu-intel-gen12=True
-          - name: E2E tests on Intel Arc A-Series Graphics
+
+          - name: E2E on Intel GEN12 Graphics; OpenCL GPU, OpenCL FPGA
+            runner: '["Linux", "gen12"]'
+            extra_lit_opts: --param gpu-intel-gen12=True
+            target_devices: opencl:gpu;opencl:fpga
+
+          - name: E2E on Intel Arc A-Series Graphics; Level zero GPU, OpenCL CPU
```

Comment:
```suggestion
          - name: E2E on Intel Arc A-Series Graphics; Level zero GPU, OpenCL GPU
```

Reply:
I'm confused. Isn't this FGPA and not GPU?

Reply:
Originally on ARC we tested 2 GPU backends: level 0 and OpenCL: `target_devices: ext_oneapi_level_zero:gpu;opencl:gpu`
Not sure, where FPGA or CPU is coming from...